src/utility/introspectionQueries.js

```diff
@@ -4,14 +4,17 @@
  *  This source code is licensed under the MIT license found in the
  *  LICENSE file in the root directory of this source tree.
  */
+import { getOperationAST, parse, introspectionQuery } from 'graphql';
 
 export { introspectionQuery } from 'graphql';
+export const introspectionQueryName = getOperationAST(parse(introspectionQuery))
+  .name.value;
```

Comment:
Can we guarantee that the `introspectionQuery` from `graphql` will always be named, @IvanGoncharov?

If not, we should add a fallback here:

```suggestion
  .name.value || "IntrospectionQuery";
```